### DISCUSSION TITLE
Distinguishing types of flows

### DISCUSSION THREAD

**Discussion by zahirb** [FlowmapBlue/FlowmapBlue#76] [General]
Hi. Thanks for creating this tool. Very cool. 
Is there a way to distinguish types of flows? For example, I have flows that are by Truck and flows that are by Rail. I can do two maps, but also nice to see where truck and rail meet on a single map. Was trying to use colors where Rail are in shades of Red while truck are in shades of Blue? Or style where Rail are dashed lines and truck are solid?
Thanks.

**Comment by ilyabo** [comment]
It's experimental, but you can add a `color` column to the flows table, like in [this spreadsheet](https://docs.google.com/spreadsheets/d/1Nf5LqbBJXOhBLqbS8IVLm1lQV--p7m20DP__5U2uSKw) or [this flowmap](https://flowmap.blue/1Nf5LqbBJXOhBLqbS8IVLm1lQV--p7m20DP__5U2uSKw?v=20.316043%2C-37.069478%2C2.22%2C0%2C0&a=0&as=1&b=1&bo=75&c=1&ca=1&d=1&fe=1&lt=1&lfm=ALL&col=Default&f=50)


<img src=https://user-images.githubusercontent.com/351828/210860159-3cf3deaf-3df3-4c67-ab46-b914df868a53.png width=400>

<img src=https://user-images.githubusercontent.com/351828/210860375-7eb9ba15-ffe2-4c9f-b94f-824b184a5c61.png width=400>

**Comment by zahirb** [comment]
Hello Ilya. Apologize for the delayed reply. I just had the opportunity to
use this functionality and it worked perfectly. Thank you.

Two things that took me a little to figure out were; (1) by default the
colors only show at a certain Zoom level so I needed to turn off clustering
if I wanted to see the color in the overall view, and (2) I was not
familiar with the color codes but found https://encycolorpedia.com/ which
gave me all the info I needed.

I thought these two pointers might make it helpful to others that use the
color feature for the first time.
Thanks.
